float32 button_triangle
float32 button_circle
float32 button_square
float32 button_cross
float32 button_L1
float32 button_L2
float32 button_R1
float32 button_R2
float32 button_share
float32 button_option
float32 button_ps4

float32 button_left_right
float32 button_up_down

float32 axes_L_Hor
float32 axes_L_Ver
float32 axes_R_Hor
float32 axes_R_Ver
float32 axes_L2
float32 axes_R2